test/NavbarSpec.js

```diff
@@ -38,7 +38,7 @@ describe('<Navbar />', () => {
   });
 
   it('can have container', () => {
-    wrapper = shallow(<Navbar container />);
-    expect(wrapper.find('nav-wrapper container')).to.have.length(0);
-  });
+    wrapper = mount(<Navbar container />);
+    expect(wrapper.props().container).to.eq(true);
```

Comment:
We know the prop container is `true` because you are setting it on L41, let's check that hasClass container is true please :)

Reply:
yeah, you are right. check my recent changes. I think the issue is fixed now